css_tasks/task5/main.css

```diff
@@ -0,0 +1,283 @@
+:root {
+  --fontColor: rgb(18, 18, 18);
+  --mainThemeColor: red;
+  --secondaryThemeColor: blue;
+  --headerFont: "Cinzel", serif;
+  --mainThemeFont: "Ubuntu", sans-serif;
+  --secondaryThemeFont: "Roboto", sans-serif;
+  --articleHeaderFont: "Bitter", sans-serif;
+  --figcaptionColor: grey;
+}
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: var(--fontColor);
+}
+.container {
+  width: 80%;
+  margin: -1rem auto 0;
+}
+/**********************
+*                     *
+*     TOGGLERS        *
+*                     *
+***********************/
+.container__toggler {
+  position: fixed;
+  margin-left: 20px;
+  left: 0;
+  cursor: pointer;
+  padding: 5px;
+  border: 1px solid var(--fontColor);
+  border-radius: 5px;
+  width:100px;
+  font-family: var(--mainThemeFont);
+  font-size:.77rem;
+  text-align:center
```

Comment:
you still write css properties in completely chaotic order(you have a lot of them in chaotic order in file), try to have some and follow it for consistency reasons and always!!! put `;` after your property

```css
.container__toggler {
  position: fixed;
  left: 0;
  width:100px;
  margin-left: 20px;
  padding: 5px;
  border: 1px solid var(--fontColor);
  border-radius: 5px;
  font-family: var(--mainThemeFont);
  font-size:.77rem;
  text-align:center;
  cursor: pointer;
}
```